proposals/NNNN-atf_facade_proposal.md

```diff
@@ -0,0 +1,255 @@
+# ATF Facade
+
+* Proposal: [SDL-NNNN](NNNN-atf_facade_proposal.md)
+* Author: [Oleksandr Deriabin](https://github.com/aderiabin)
+* Status: **Awaiting review**
+* Impacted Platforms: [ATF]
+
+## Introduction
+
+This proposal describes unified interface for ATF (Automated Test Framework) - ATF Facade.
+ATF should provide unified and flexible interface for creation test scripts to perform checks of SDL functionality.
+
+Main ATF Facade purposes:
+ - Simplify access to ATF features for test scripts
+ - Separate test scripts from ATF
+ - Provide well-structured intuitive interface of ATF for test scripts
+ - Unify way of use ATF for creating automated tests
+ - Create single point of ATF features documentation
+ - Provide ability to create more human-readable tests
+
+## Motivation
+
+Currently ATF has a lot of functionality for testing most of SDL features, but each automated test uses different ATF modules.
+ATF does not have unified interface to provide its functionality for test scripts. Instead of this ATF has many separated modules with their own interfaces which are used in automated scripts for SDL testing purposes.
+
+So current approach leads to create test scripts which have multiple direct dependencies on different modules of ATF.
+In other words automated test scripts are fully dependent on ATF implementation and each change in ATF can lead to need for change of test scripts.
```

Comment:
Please update to "...can lead to a need to change test scripts."

Reply:
Updated (see 9943914522e19530a121b117461502d1961c2b27)